libp2p/src/bandwidth.rs

```diff
@@ -43,9 +43,11 @@ use std::{
 pub(crate) struct BandwidthLogging<SMInner> {
     #[pin]
     inner: SMInner,
+    #[allow(deprecated)]
```

Comment:
I think it is easier to just place this on the entire module as we will just delete this, right?

I.e. use `#![allow(deprecated)]` at the top.

Reply:
Sounds good. Thank you for the code snippet.